### DISCUSSION TITLE
SKCC nr & POWER field in Log Entry Window, How to make this work

### DISCUSSION THREAD

**Discussion by PE1FJN** [magicbug/Cloudlog#2553] [General]
Hello Cloudlog Community,

I would like to log my TX power per QSO in the log entry window and also SKCC number from SKCC stations.

Is this possible to organize?  This will help me to log SKCC stations and log QRP power levels as part of the BENELUX QRPC Club so-called marathon, where each QSO can run on different power levels from 100mW e.g. ... 5W, the lower the power the more points per QSO, hi.

I am running/using Cloudlog as a service from MAGICBUG, and have ZERO coding capabilities.

Does anyone have suggestions?

73
PE1FJN
Marc

**Comment by phl0** [comment]
Have a look at the "station" tab on the QSO entry view.

**Reply by PE1FJN** [reply]
TNX Power Field Solved :).....

**Comment by magicbug** [comment]
To track SKCC members and numbers you can use the Awards SIG feature

When saving a QSO under the General tab set Sig field to "SKCC" and Sig Info to what ever number you require.

You can then track this via Awards -> SIG and you should see SKCC all listed together

**Reply by PE1FJN** [reply]
Sorry Late reply, works for me, 73 es TNX, Marc
